_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,92 @@
+/* ============================
+-------------------------------
+ 1. Basic Setup
+-------------------------------
+============================ */
+
+
+/* ============================
+1. Basic Setup
+---------------------------- */
+html {
+    height: 102%;
```

Comment:
This was pre-existing. The comment said that it forced a scrollbar and that the overflow settings of yesteryear were problematic with positioning. Or something like that. I've never used this before though, so am happy to take it off.